bazel_example/gapic-generator-cloud/BUILD.bazel

```diff
@@ -42,18 +45,18 @@ ruby_gapic_assembly_pkg(
 ##
 # just the gapic-generator-ruby cloud flavor output
 #
-ruby_gapic_cloud_library(
+ruby_cloud_gapic_library(
   name = "language_ruby_gapic",
   srcs = ["@com_google_googleapis//google/cloud/language/v1:language_proto_with_info"],
-  ruby_cloud_params = {
-    ":gem.:name": "google-cloud-language-v1",
-    ":gem.:title": "Natural Language V1",
-    ":gem.:description": "Provides natural language understanding technologies, such as sentiment analysis, entity recognition, entity sentiment analysis, and other text annotations.",
-    ":gem.:env_prefix": "LANGUAGE",
-    ":gem.:product_documentation_url": "https://cloud.google.com/natural-language",
-    ":gem.:api_id": "language.googleapis.com",
-    ":gem.:api_shortname": "language",
-  },
+  extra_protoc_parameters = [
+    ":gem.:name = google-cloud-language-v1",
```

Comment:
I would be inclined not to include spaces around the `=` delimiter, so we don't have to strip() when parsing, and can avoid some ambiguity.

Reply:
Right, that is probably too cute when I think about it.